.circleci/config.yml

```diff
@@ -8,7 +8,8 @@ jobs:
   checkout_code:
     working_directory: ~/search-gov
     docker:
-      - image: circleci/ruby:2.3.5-browsers
+      # using custom image, see .circleci/images/primary/Dockerfile
+      - image: mothra/search.gov:011119
```

Comment:
Note: At the moment, this image is in my personal Docker Hub account. I've created a follow-up story to move the image to an official, shared Docker Hub organization, which Dawn can set up when she returns.